lib/ffmpeg/transcoder.rb

```diff
@@ -204,6 +211,14 @@ def transcode_movie
           raise Error, "Process hung. Full output: #{@output}"
         end
       end
+      if File.exist?(temp_output_file)
+        FileUtils.cp(temp_output_file, @output_file)
+        FileUtils.rm_rf(temp_output_file)
+      end
```

Comment:
Should we add a comment somewhere in the file why we're doing these temp operations (Specifically due to mountpoint incompatibilities) incase things change when we migrate to EKS and can remove this?

Reply:
I mean technically the git blame does point to the PR for the changes? I guess at a glance - But i could see it being maybe problematic if its left?  I feel making note in the PR regarding those incompatibilities would be more ideal? but I am not sure @Kyrluckechuck ?

Reply:
Sure, as long as it's a concise comment I have no issue with it. Could also add the comment to the top where we create the temp files saying why it's done there and then add the comment here just saying it's cleaning up the temp files